polytope/polytope.py

```diff
@@ -290,6 +290,32 @@ def intersect(self, other, abs_tol=ABS_TOL):
 
         return reduce(Polytope(iA, ib), abs_tol=abs_tol)
 
+    def translation(self, d):
+        """Translate the Polytope by a given vector.
```

Comment:
I agree with all of the comments and recommendations above, but I did not consider it required before this PR would be accepted because the meaning of "vector" can be inferred. Instead, we might return to docstrings later during quality assurance review of documentation.

Alternatively, the docstring that @johnyf pitches above is good for me, too.